pyomo/contrib/pynumero/examples/structured/tests/test_nlp_compositions.py

```diff
@@ -131,9 +131,8 @@ def setUpClass(cls):
             scenario_name = "s{}".format(i)
```

Comment:
This file will disappear with #1936 . I'm surprised these changes needed to happen since these tests do not run or work.

Reply:
These changes were not strictly necessary, but I was testing things with `pytest` (which picks up more tests than `nosetests`) and these errors were being highlighted.  Unfortunately, after fixing these, it became obvious that there were going to be more substantial changes needed to get things running, so I stopped pulling this thread.  It probably shouldn't have been in this PR at all.

Reply:
Thanks for the explanation. That makes sense.